src/cohere/client.py

```diff
@@ -97,6 +101,44 @@ def __exit__(self, exc_type, exc_value, traceback):
 
     wait = wait
 
+    _executor = ThreadPoolExecutor()
+
+    def embed(
```

Comment:
Can we add a docstring so that help(co.embed) shows all the parameters with descriptions

Reply:
fortunately the docs get inherited from the autogenerated base class so we can leave off here 
<img width="1127" alt="image" src="https://github.com/cohere-ai/cohere-python/assets/144115527/7f950388-f67b-418e-a38b-8c518195c6ee">